app/src/main/java/com/concordium/wallet/data/preferences/Preferences.kt

```diff
@@ -50,6 +50,16 @@ open class Preferences {
         triggerChangeEvent(key)
     }
 
+    protected fun setStringWithResult(key: String, value: String?): Boolean {
+        val editor = editor
```

Comment:
Editing takes time, so we create a local instance that will perform the edit. If we use the global editor, he can be called from a different function at the same time and we could end up with errors or committing the same changes multiple times.

Reply:
This could be a little bit better named( the global and local have the same name), but thats how it's written before in that whole class.
The local "val editor" is a copy of the global "editor"
The local "editor" overrides the global "editor" inside the function and all of the action is performed with the local editor.